### ISSUE TITLE
Gradient coloring for Lines-mark

### ISSUE THREAD

**Issue by alha02** [bloomberg/bqplot#996]
Would it be possible to do gradient coloring for lines-mark, such as e.g. the corresponding functionality available in Excel?

**Comment by martinRenou** [comment]
Could it be flexline that you are looking for? See https://github.com/bloomberg/bqplot/blob/master/examples/Marks/Object%20Model/FlexLine.ipynb

**Comment by alha02** [comment]
Yes, that’ the type of thing I was looking for, thanks. Ideally though, I would have wanted gradient coloring in between datapoints too, is that possible to do with flexline?

**Comment by martinRenou** [comment]
I am not sure... I feel like the only thing you can do is to have more data points. I wonder if that could be easily added to the flexline implementation though.
